plugin/src/test/java/io/jenkins/plugins/casc/ConfigurationAsCodeTest.java

```diff
@@ -7,15 +7,17 @@
 import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
 import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
 import java.io.File;
+import java.io.InputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
 import java.util.List;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.*;
```

Comment:
We don't like star imports :sweat:

Reply:
Sorry, my IDE rules are doing this automatically, will fix.

Reply:
You can set project specific rules usually :sweat_smile: